src/plugins/felixhayashi/tiddlymap/tiddlers/misc/misc.config.edit_template_fields_visibility.multids

```diff
@@ -0,0 +1,3 @@
+title: $:/config/EditTemplateFields/Visibility/
```

Comment:
:thinking:  shouldn't the content of this tiddler be

```
$:/config/EditTemplateFields/Visibility/tmap.id

hide
```

e.g. compare: `$:/config/EditTemplateFields/Visibility/creator` or `$:/config/EditTemplateFields/Visibility/modifier`.

Reply:
I'm using a .multids file for this one. They're better suited for single line tiddlers like this. If you look at tiddlywiki source, they actually use a multids file for all of the EditTemplateFields/Visibility settings.
```
title: $:/config/EditTemplateFields/Visibility/
  
title: hide
tags: hide
text: hide
creator: hide
created: hide
modified: hide
modifier: hide
type: hide
draft.title: hide
draft.of: hide
revision: hide
bag: hide
```
Even for a single value, they're nice because they can't accidentally include newline characters, which can be problematic (especially for vi users like me).

Reply:
Ah okay, I wasn't sure if in case two plugins are having a multids tiddler with `title: $:/config/EditTemplateFields/Visibility/` whether this would result in one plugin overriding the shadow tiddler of the other. Therefore I thought having a tiddler with a title `$:/config/EditTemplateFields/Visibility/tmap.id` would be safer as it exclusively belongs to tiddlymap unless being overridden by the person who installed the plugin. But it seems during import multids are split into separate tiddlers so I am fine with this solution.